packages/@angular/cli/blueprints/ng/files/__path__/index.html

```diff
@@ -9,6 +9,6 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 </head>
 <body>
-  <<%= prefix %>-root>Loading...</<%= prefix %>-root>
+  <<%= prefix %>-root></<%= prefix %>-root>
```

Comment:
It conveys a message that it will take a while for Angular to load which is just not true so the decision was made to remove it by default... if you'd like to add it in you certainly can.